docs/kpod-images.1.md

```diff
@@ -1,15 +1,17 @@
+% kpod(1) kpod-images - List images in local storage.
+% Dan Walsh
 ## kpod-images "1" "March 2017" "kpod"
 
-## NAME
-kpod images - List images in local storage.
+# NAME
+kpod images - 
```

Comment:
I'd put the "List images in local storage." back here, but if you think it's best to be left off, please remove the space and '-' at the end of the line